policy/scan/local_scanner.go

```diff
@@ -384,6 +384,16 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 						asset.Mrn = details.AssetMrn
 						asset.Url = details.Url
 						asset.Labels["mondoo.com/project-id"] = details.ProjectId
+
+						if asset.Annotations == nil {
+							asset.Annotations = make(map[string]string)
+						}
+						for k, v := range details.Annotations {
+							if _, ok := asset.Annotations[k]; !ok {
```

Comment:
I couldn't find details about this.
It is intended that client-side annotations take precedence?

Reply:
Yes, if its set on the client it takes precedence